CODE-GUIDELINES.md

```diff
@@ -595,6 +598,100 @@ Never use `!important` to raise the specificity of a rule. In well architected C
 }
 ```
 
+
+
+<!-- ============================================================ -->
+
+## Spaces
+
+Always use PatternFly spacing variables to define `margin` and `padding`
+
+```sass
+$pf-spacer-xxxs   // ~3px
+$pf-spacer-xxs    // 7px
+$pf-spacer-xs     // ~10px
+$pf-spacer-sm     // 14px
+$pf-spacer-md     // ~17px
+$pf-spacer-lg     // 21px
+$pf-spacer-xl     // ~24px
+$pf-spacer-xxl    // 28 px
+$pf-spacer-xxxl   // 35 px
+$pf-spacer-xxxxl  // 42 px
+```
+
+```sass
+.nav-link {
+  padding-bottom: $pf-spacer-xxxs;
+}
+
+.dropdown-menu {
+  margin-left: $pf-spacer-sm;
+}
+```
+
+You can also use [Bootstrap spacing utilities](http://v4-alpha.getbootstrap.com/utilities/spacing/) with PatternFly sizes:
+
+```html
+<p class="mb-lg">This paragraph has a large bottom margin</p>
+```
+
+<!-- ============================================================ -->
+
+## Shadows
+
+PatternFly has 4 levels of shadows, always variables to apply `box-shadow`:
```

Comment:
Maybe just say there are 6 types? Is the term "levels" used elsewhere (besides in the template)?